api/src/zenbox_smt/bitvector_smt.rs

```diff
@@ -0,0 +1,192 @@
+////////////BITVECTOR SPARSE MERKLE TREE
+
+use primitives::{BigMerkleTree, Coord};
+use crate::ginger_calls::{GingerMerkleTreeParameters, Error, FieldElement, GingerMHTPath, GingerSMT, restore_ginger_smt, new_ginger_smt};
+use algebra::fields::mnt6753::FqParameters;
+use algebra::{FpParameters, ToBits};
+use std::path::Path;
+
+pub type BitvectorSMT = BigMerkleTree<GingerMerkleTreeParameters>;
```

Comment:
@nastenko 
Introduce a function to serialize the Bitvector leaves to a byte array. The serialization should avoid to include the fieldelements bits not available in leaves. 
For example, with field modulus size 255 bits, the available bits are 254 and serialized size must be 254 bits and not 256 bits. This means that if we have to serialize 8 field elements the serialize size in bytes must 254 bytes instead of 256.